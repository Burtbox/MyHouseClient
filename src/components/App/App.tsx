import * as React from 'react';

import Nav from '../Nav';
import Routes from '../Routes';
import ErrorMessage from '../ErrorMessage';
import Footer from '../Footer';

function App() {
    return (
        <div style={{ height: '100%' }}>
            <Nav />
            <Routes />
            <ErrorMessage />
            <Footer />
        </div>
    );
}

export default App;
