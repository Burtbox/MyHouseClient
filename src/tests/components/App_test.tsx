import * as React from 'react';
import { shallow } from 'enzyme';
import App from '../../components/App';

describe('App test suite', () => {
    test('Render app component', () => {
        const AppScreen = shallow(
            <App />,
        );
        expect(AppScreen).toMatchSnapshot();
    });
});
