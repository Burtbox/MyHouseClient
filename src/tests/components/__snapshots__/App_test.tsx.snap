// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test suite Render app component 1`] = `
<div
  style={
    Object {
      "height": "100%",
    }
  }
>
  <Connect(Nav) />
  <withRouter(Connect(Routes)) />
  <Connect(ErrorMessage) />
  <Footer />
</div>
`;
